core/config/project_settings.cpp

```diff
@@ -1241,6 +1241,17 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF("input/ui_end", action);
 	input_presets.push_back("input/ui_end");
 
+	action = Dictionary();
+	action["deadzone"] = Variant(0.5f);
+	events = Array();
+	key.instance();
+	key->set_keycode(KEY_ENTER);
+	key->set_alt(true);
+	events.push_back(key);
```

Comment:
```suggestion
	events.push_back(key);
	key.instance();
	key->set_keycode(KEY_F11);
	events.push_back(key);
```

Reply:
You can use the new `InputEventKey::create_instance( KEY_MASK_ALT | KEY_ENTER)` here to keep it succinct.